### PR TITLE
Add git_create_release script

### PR DESCRIPTION
**Pull request by wahlg**
A script for creating a release PR in github. Accepts a -t  or --time-of-day argument which can be 'morning', 'afternoon', or 'evening', and then creates XML info in the PR based on that date.

### CODE REVIEW

File: lib/create_release.py [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+  print('-- BODY --')
+  print(body)
+
+def create_release(args):
+  git_scripts_repo_path = os.path.dirname(os.path.dirname(os.path.realpath(__file__)))
+  h2 = Helper2(git_scripts_repo_path)
+  branch = h2.base_branch()
+  release_branch = 'release'
+  date = get_date_from_args(args)
+  owner, repo = Helper.owner_and_repo()
```
Comment (santi-h) [inline lib/create_release.py]: I think `h2` has its own version of `owner_and_repo`, `Helper` is deprecated we should use `Helper2`

File: lib/create_release.py [hunk 2]
```diff
@@ -0,0 +1,101 @@
+from misc import Helper
+from misc.helper2 import Helper2
+from misc.helper2 import get_script_name_from_python_file
+from misc.utils import run_command_and_exit, say
+from lxml.etree import Element, SubElement, tostring
+from create_pr import ObjectNotFoundException
```
Comment (santi-h) [inline lib/create_release.py]: `ObjectNotFoundException` is not used in this file?

File: lib/create_release.py [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+def get_time_from_config(time_of_day):
+  h2 = Helper2()
+  release_config = h2.yaml('release-time')
+  if release_config is None:
+    say("There's no release configuration specified for this repository.")
+    sys.exit(1)
+
+  time = release_config.get(time_of_day)
+  if not time:
+    raise InvalidTimeOfDayException('Time of day must be \'morning\', \'afternoon\', or \'evening\'')
```
Comment (santi-h) [inline lib/create_release.py]: Does the user see a nice one liner for this or an stacktrace exeption or a "want to report this issue?" message? we shouldn't prompt for reporting it

File: lib/create_release.py (hunk 1 above)
Comment (wahlg) [reply lib/create_release.py]: I've changed it to use h2's version instead

File: lib/create_release.py (hunk 2 above)
Comment (wahlg) [reply lib/create_release.py]: That was left in by mistake

File: lib/create_release.py (hunk 3 above)
Comment (wahlg) [reply lib/create_release.py]: They would have seen the "want to report this issue?" message. I've changed it so it doesn't prompt and exits the same way it does if no configuration is specified.

**Comment by wahlg** [comment]
Closing this for now. See https://github.com/bodyshopbidsdotcom/git-scripts/pull/335 for a simplified version of this script. We may add these features in later.

**Pull request closed without merging**